packages/suite/src/components/suite/DeviceDisplay.tsx

```diff
@@ -0,0 +1,230 @@
+`;
+
+const Row = styled.div<{ isPixelType: boolean; isAlignedRight?: boolean }>`
+    display: flex;
+    align-items: center;
+    gap: ${props => (props.isPixelType ? '8' : '9')}px;
+    justify-content: ${props => (props.isAlignedRight ? 'flex-end' : 'flex-start')};
+`;
+
+const Chunks = styled.div<{ isPixelType: boolean }>`
+    display: flex;
+    flex-direction: column;
+    gap: ${props => (props.isPixelType ? '6' : '5')}px;
+`;
+
+const Wrapper = styled.div`
+    position: relative;
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+`;
+
+const Divider = styled.div<{ isPixelType: boolean }>`
+    width: 100%;
+    height: 1px;
+    background: #2b2b2b;
+    margin: ${props => (props.isPixelType ? '10px' : '0')} 0;
+`;
+
+const AddressLabel = styled.span<{ isPixelType: boolean }>`
+    font-weight: 600;
+    color: #868686;
+    font-size: ${variables.FONT_SIZE.TINY};
+    text-transform: uppercase;
+    position: absolute;
+    background: #000000;
+    padding: 0 10px;
+    text-align: center;
+    left: 50%;
+    bottom: ${props => (props.isPixelType ? '2' : '-7')}px;
+    transform: translate(-50%, 0);
+`;
+
+const StyledNextIcon = styled(Icon)<{ isPixelType: boolean }>`
+    position: relative;
+    bottom: ${props => (props.isPixelType ? '13' : '20')}px;
+    right: ${props => (props.isPixelType ? '-95' : '35')}px;
+`;
+
+const StyledContinuesIcon = styled(Icon)<{ isPixelType: boolean }>`
+    position: relative;
+    top: ${props => (props.isPixelType ? '10' : '25')}px;
+    right: ${props => (props.isPixelType ? '82' : '97')}px;
+`;
+
+const StyledChunkedNextIcon = styled(Icon)<{ isPixelType: boolean }>`
+    position: relative;
+    bottom: ${props => (props.isPixelType ? '13' : '20')}px;
+    right: -67px;
+`;
+
+const StyledChunkedContinuesIcon = styled(Icon)<{ isPixelType: boolean }>`
+    position: relative;
+    top: ${props => (props.isPixelType ? '10' : '25')}px;
+    right: 68px;
+`;
+
+export interface DeviceDisplayProps {
+    address: string;
+    network?: string;
+}
+
+export const DeviceDisplay = ({ address, network }: DeviceDisplayProps) => {
+    const selectedDeviceInternalModel = useSelector(selectDeviceInternalModel);
+    const unavailableCapabilities = useSelector(selectDeviceUnavailableCapabilities);
+    const addressDisplay = useSelector(selectAddressDisplay);
+
+    const areChunksUsed =
+        addressDisplay === AddressDisplayOptions.CHUNKED && !unavailableCapabilities?.chunkify;
+    const isPixelType = selectedDeviceInternalModel !== DeviceModelInternal.T2T1;
+    const isPaginated = network === 'cardano';
+
+    const showChunksInRows = (chunks: string[][] | undefined, isNextAddress?: boolean) =>
+        chunks?.map((row, rowIndex) => (
+            <Row
+                // eslint-disable-next-line react/no-array-index-key
+                key={rowIndex}
+                isPixelType={isPixelType}
+                isAlignedRight={rowIndex === 0 && isNextAddress}
```

Comment:
not needed if `grid is used`?

Reply:
The line number 4 was render not fully to the right so I have to do it like this

Reply:
Almost, grid will solve the row alignment but still I need to align somehow the the second icon to the left. Will keep it propbably with `flex`.

Reply:
You mean to the right? You can use `justify-self` on the icon and have this in all the rows

```
    display: grid;
    grid-template-columns: 1fr 1fr 1fr 1fr;
```

Reply:
For some reason it doesn't work, let's keep the flex